Sources/Bugsnag/PayloadTransformer.swift

```diff
@@ -3,17 +3,25 @@ import Stacked
 import HTTP
 
 public protocol PayloadTransformerType {
-    func payloadFor(message: String, metadata: Node?, request: Request?) throws -> JSON
+    func payloadFor(message: String, metadata: Node?, request: Request?, filters: [String]) throws -> JSON
 }
 
 internal struct PayloadTransformer: PayloadTransformerType {
     let drop: Droplet
     let config: ConfigurationType
+    let filters: [String]
+
+    init(drop: Droplet, config: ConfigurationType) {
+        self.drop = drop
+        self.config = config
+        self.filters = config.filters
```

Comment:
See my other comment, consider only using the filters passed in the `payloadFor..` function.